src/Microsoft.AspNetCore.SignalR.Client/HubConnection.cs

```diff
@@ -45,29 +46,29 @@ public class HubConnection
             remove { _connection.Closed -= value; }
         }
 
+        public HubConnection(Uri url, IInvocationAdapter adapter)
```

Comment:
We need to fix it. This isn't maintainable and these scenarios don't all have equal importance.